build.gradle.kts

```diff
@@ -305,6 +305,23 @@ task("python38PostCommit") {
   dependsOn(":sdks:python:test-suites:portable:py38:postCommitPy38")
 }
 
+
+task("pythonExamplesFlinkPostCommit") {
+  dependsOn(":sdks:python:test-suites:portable:py37:postCommitExamplesFlink")
```

Comment:
let's try to not hardcode a particular python version here and instead specify them in one place, see examples in 

https://github.com/apache/beam/blob/879048179ac9976275da1bd4f59a30ca49dd66af/sdks/python/test-suites/portable/build.gradle#L29

https://github.com/apache/beam/blob/879048179ac9976275da1bd4f59a30ca49dd66af/sdks/python/test-suites/gradle.properties

Reply:
Fixed!